examples/TicTacToe-Console.links

```diff
@@ -21,11 +20,11 @@ fun playerToString(player) {
     }
 }
 
-sig optionToString : ((a) -e-> String) -> (Option(a)) -e-> String
+sig optionToString : ((a) -e-> String) -> (Maybe(a)) -e-> String
```

Comment:
`maybeToString`?

Reply:
OK, though possibly this could be a library function, it is a special case of the "fold" (or "case") operation for maybe:
```
foldMaybe :: (Maybe(a),b,a -e-> b) -e-> b
```

Reply:
It certainly shouldn't be called `optionToString` any more!

Reply:
https://github.com/links-lang/links/pull/1131/commits/6cffb2d68915e733fa243963b12d73312823f5b4